.github/workflows/self-scheduled-amd-caller.yml

```diff
@@ -5,7 +5,7 @@ on:
     - cron: "17 2 * * *"
   push:
     branches:
-      - run_amd_scheduled_ci_caller*
+      - run_amd_scheduled_ci_caller__*
```

Comment:
will remove this modification before merging (added to disable all the other AMD scheduled tests)